examples/src/examples/FiInteractiveEngineExample.java

```diff
@@ -63,13 +63,24 @@ public static void main(String[] args) throws InterruptedException, JsonProcessi
       FiCalculationsApi apiInstance = new FiCalculationsApi(getApiClient());
       FICalculationParameters calcParameters = new FICalculationParameters();
       
+      FIBankLoans fibankloans = new FIBankLoans();
+      fibankloans.ignoreSinkingFund(true);
+      
+      FIMunicipalBonds fimunicipalbonds = new FIMunicipalBonds();
+      fimunicipalbonds.ignoreSinkingFund(true);
+      
+      FIMunicipalBondsForJobSettings fimunicipalbondsforjobsettings = new FIMunicipalBondsForJobSettings();
+      fimunicipalbondsforjobsettings.ignoreSinkingFund(true);
```

Comment:
Very minor: Can you please move these lines to 104 as they are specific to only job settings?

Reply:
Updated